dts/bindings/pwm/microchip,xec-pwmbbled.yaml

```diff
@@ -46,6 +46,13 @@ properties:
   "#pwm-cells":
     const: 2
 
+  enable-low-power-32k:
+    type: boolean
+    description: |
+      Used to gates off 32K clock and turn off LEDs during sleep.
+      This applicable only to 32K clock, because in 48MHz clock
+      will be always turned off in low power by PCR.
```

Comment:
Can we improve the description a bit? From a user point of view, I find it very difficult to parse the relationship between this property, the power management subsystem, and the input clock.

Reply:
@henrikbrixandersen Is below ok?

BBLED has two clock inputs.
 1. Main system clock (48MHz)
 2. 32KHz Core clock  (32.768KHz)
When BBLED enter into Suspend state, 48MHz clock
will be switched off by PCR(Power, Clock and Reset) block.
But 32KHz Core clock will be available to BBLED.

There may be a product requirement, either to blink (or) not blink LED
in Suspend state.

"enable-low-power-32k" flag shall be used along with 32KHz clock to blink (or) not blink the LED in Suspend state.

Reply:
Yes, if you could update the description to something along those lines, I think will be a huge improvement.

Reply:
Updated the description